packages/generator/src/generator.js

```diff
@@ -47,6 +54,7 @@ export default class Generator {
 
     // Done hook
     await this.nuxt.callHook('generate:done', this, errors)
+    await this.nuxt.callHook('export:done', this, { errors })
```

Comment:
Are we gonna deprecate `generate:*` hooks ?

Reply:
Hmm not sure. But since it is for `nuxt export` and we already have `export` config alias, was thinking makes sense to introduce new hooks so we have backward compatibility if deprecating for Nuxt3

Reply:
Indeed we duplicate them now, and we will deprecate nuxt generate (I think we can for Nuxt 2.14) and delete it for Nuxt 3

Reply:
We have to be careful since it’s not only renaming for module authors :)

Reply:
Can we find a nice way for module author to give the option that export is available? Without needing to check the Nuxt version?

Reply:
@Atinux They can check `options.target` to see if using `static` target or not. Also agree about deprecate if we also deprecate `nuxt generate` command.

Reply:
Perfect 🙌